docs/download_and_install.md

```diff
@@ -32,8 +46,3 @@ Copy the link shown by pwd and add /index.html in a web browser
 cmake -D CMAKE_CXX_COMPILER=/usr/local/bin/g++-11 CMAKE_C_COMPILER=/usr/local/bin/gcc-11 ..
 ```
 **NOTE**: Replace g++-11 and gcc-11 with the compilers you desire
-
```

Comment:
We really need to discuss this option OPENMP

Reply:
My thinking is that currently it is required, so we should advertise it as such. To be fair pretty much every modern compiler anyway supports openmp, so this is asking for a lot.
Once it becomes truly optional it's not hard to add these checks back in, but for now I think it's misleading in the docs and boilerplate in the CMakeLists.txt.